dask/dataframe/tests/test_io.py

```diff
@@ -547,11 +547,10 @@ def test_read_csv_of_modified_file_has_different_name():
     with filetext(text) as fn:
         mtime = os.path.getmtime(fn)
         a = read_csv(fn)
+        sleep(1)
         with open(fn, 'a') as f:
             f.write('\nGeorge,700')
             os.fsync(f)
-        while mtime == os.path.getmtime(fn):
-            sleep(0.01)
```

Comment:
If I understand correctly, Osx's timing function only has 1 sec resolution for filesystem information. So we read in `a`, sleep a second, then modify the file and read in `b`. If we did it the other way, both the initial creation and modification happen within the same 1-sec bin, and the test never exits.